ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/PicoCliDownstreamTlsClientAuthOptions.java

```diff
@@ -18,20 +18,20 @@
 
 import picocli.CommandLine.Option;
 
-public class PicoCliTlsClientCertificateOptions implements PkcsStoreConfig {
+public class PicoCliDownstreamTlsClientAuthOptions implements PkcsStoreConfig {
 
   @Option(
       names = "--downstream-http-tls-keystore-file",
       description =
-          "Path to a PKCS#12 formatted keystore, contains TLS certificate to present to "
-              + "a TLS-enabled web3 provider.",
+          "Path to a PKCS#12 formatted keystore, contains key/certificate to present to "
```

Comment:
Does the PKCS file need to contain a key? Or just a certificate?

Reply:
I this particular option, both. Only when PKCS is used as a "trust store" it can take public certificate. Here the client needs to set its keystore (which takes both certificate and private key).